HealthcareSystem/Backend/Repository/DrugInRoomRepository/MySqlDrugInRoomRepository/MySqlDrugInRoomRepository.cs

```diff
@@ -42,7 +42,7 @@ public DrugInRoom SetDrug(DrugInRoom drugInRoom)
 
         public DrugInRoom GetDrugInRoomByDrugId(int drugId)
         {
-            return (DrugInRoom)_context.DrugsInRooms.Where(x => x.DrugId == drugId).ToList()[0];
+            return _context.DrugsInRooms.Single(x => x.DrugId == drugId);
```

Comment:
Why would there be just one though? Can't the same drug be distributed in multiple rooms?

Reply:
Fixed in 28738c042abebb9f51d7934846677a2f4a969db6.